stackslib/src/chainstate/stacks/transaction.rs

```diff
@@ -951,6 +986,98 @@ impl StacksTransaction {
         Ok(next_sighash)
     }
 
+    pub fn sign_no_append_origin(
```

Comment:
It looks like this and the next 3 functions are only used in test code. Can you add `#[cfg(test)]` to them?